src/mbgl/style/expression/distance.cpp

```diff
@@ -33,10 +34,39 @@ std::size_t getRangeSize(const IndexRange& range) {
     return range.second - range.first + 1;
 }
 
+bool isRangeSafe(const IndexRange& range, const std::size_t threshold) {
+    return (range.second >= range.first && range.second < threshold);
+}
+
+std::pair<mbgl::optional<IndexRange>, mbgl::optional<IndexRange>> splitRange(const IndexRange& range, bool isLine) {
+    if (range.first > range.second) return std::make_pair(nullopt, nullopt);
+    auto size = getRangeSize(range);
+    if (isLine) {
+        if (size == 2) {
```

Comment:
is it possible `size < 2`?

Reply:
After adding the size checker before calculation, for lines, the minimum size will be 2.